docs/index.md

```diff
@@ -0,0 +1,93 @@
+# Ops documentation
+
+```{toctree}
+:maxdepth: 2
+:hidden: true
+
+tutorial/index
+howto/index
+reference/index
+explanation/index
+```
+
+
+
+The Ops library (`ops`) is a Python framework for writing and testing Juju charms.
+
+> [See it on PyPI](https://pypi.org/project/ops/)
+
+The library provides:
+
+- {ref}`ops_main_entry_point`, used to initialise and run your charm
+- {ref}`ops`, the API to respond to Juju events and manage the application
+- {ref}`ops_pebble`, the Pebble client, a low-level API for Kubernetes containers
+- {ref}`ops_testing`, the recommended API for unit testing charms
+- {ref}`ops_testing_harness`, the deprecated API for unit testing charms
+
+You can structure your charm however you like, but with the `ops` library, you get a framework that promotes consistency and readability by following best practices. It also helps you organise your code better by separating different aspects of the charm, such as managing the application's state, handling integrations with other services, and making the charm easier to test.
+
+
+---------
+
+## In this documentation
+
+````{grid} 1 1 2 2
+
+```{grid-item-card} [Tutorial](tutorial/index)
+:link: tutorial/index
+:link-type: doc
+
+**Start here**: a hands-on introduction to Example Product for new users
+```
+
+```{grid-item-card} [How-to guides](/index)
+:link: howto/index
+:link-type: doc
+
+**Step-by-step guides** covering key operations and common tasks
+```
+
+````
+
+
+````{grid} 1 1 2 2
+:reverse:
+
+```{grid-item-card} [Reference](/index)
+:link: reference/index
+:link-type: doc
+
+**Technical information** - specifications, APIs, architecture
+```
+
+```{grid-item-card} [Explanation](/index)
+:link: explanation/index
+:link-type: doc
+
+**Discussion and clarification** of key topics
+```
+
+````
+
+
+---------
+
+
+## Project and community
+
+Ops is a member of the Ubuntu family. It’s an open source project that warmly welcomes community projects, contributions, suggestions, fixes and constructive feedback.
+
+* **[Read our code of conduct](https://ubuntu.com/community/ethos/code-of-conduct)**:
+As a community we adhere to the Ubuntu code of conduct.
+
+* **[Get support](https://discourse.charmhub.io/)**:
+Discourse is the go-to forum for all questions Ops.
+
+* **[Join our online chat](https://matrix.to/#/#charmhub-ops:ubuntu.com)**:
+Meet us in the #charmhub-charmdev channel on Matrix.
```

Comment:
TIL that the link to the "Ops Library Development" room is "#charmhub-ops". I probably caused this change with an error comment, sorry, but we're now inconsistent.

Note that the footer (from docs/custom_conf.py) links to the Charm Development channel. It's a bit odd having "Join our online chat" on the landing page be to Ops Library Development and the footer on every page having an "Ask a question on Matrix" link that goes to Charm Development.

My feeling is that most of the time we'd get questions about *writing and testing charms*, so Charm Development is the best place. If people do have a question about *developing and maintaining ops itself* and they posted in Charm Development, I don't think we'd miss it, and it'd be less likely.

So my suggestion is:

```suggestion
* **[Join our online chat](https://matrix.to/#/#charmhub-charmdev:ubuntu.com)**:
Meet us in the Charm Development (#charmhub-charmdev) channel on Matrix.
```

Reply:
I can do this in a follow-up PR